lasagne/layers/special.py

```diff
@@ -796,13 +842,16 @@ def _initialize_tps(num_control_points):
 
     # Create 2 x num_points array of source points
     source_points = np.vstack(
-            (x_control_source.flatten(), y_control_source.flatten()))
+        (x_control_source.flatten(), y_control_source.flatten()))
```

Comment:
I'd prefer double indent as before (make sure to use the same pep8 version as in `requirements-dev.txt`).